apps/dashboard/src/contract-ui/tabs/permissions/components/permissions-editor.tsx

```diff
@@ -193,14 +191,7 @@ const PermissionAddress: React.FC<PermissionAddressProps> = ({
                 e.stopPropagation();
                 e.preventDefault();
                 onCopy();
-                toast({
-                  position: "bottom",
-                  variant: "solid",
-                  title: "Address copied.",
-                  status: "success",
-                  duration: 5000,
-                  isClosable: true,
-                });
+                toast.success("Address copied.");
```

Comment:
This shouldn't be "success" variant - if there is other visual feedback for copy action - we can completely drop this - otherwise use "info" variant